lldb/tools/lldb-dap/package.json

```diff
@@ -53,9 +55,10 @@
     "bundle-symbols-table-view": "npx tsc -p src-ts/webview --noEmit && npx esbuild src-ts/webview/symbols-table-view.ts --bundle --format=iife --outdir=./out/webview",
     "bundle-tabulator": "cp node_modules/tabulator-tables/dist/js/tabulator.min.js ./out/webview/ && cp node_modules/tabulator-tables/dist/css/tabulator_midnight.min.css ./out/webview/ && cp node_modules/tabulator-tables/dist/css/tabulator_simple.min.css ./out/webview/",
     "bundle-webview": "npm run bundle-symbols-table-view && npm run bundle-tabulator",
+    "check-unit": "npx tsc -p ./ && mocha ./out/unittests-ts",
```

Comment:
If we add `@vscode/test-cli` to the deps we could use `vscode-test` here instead.

https://github.com/microsoft/vscode-test-cli for reference.